src/main/kotlin/com/kotlin/boardproject/domain/auth/AuthController.kt

```diff
@@ -1,28 +1,28 @@
-package com.kotlin.boardproject.controller
-
-import com.kotlin.boardproject.common.config.properties.AppProperties
-import com.kotlin.boardproject.common.util.addCookie
-import com.kotlin.boardproject.common.util.deleteCookie
-import com.kotlin.boardproject.dto.TokenResponseDto
-import com.kotlin.boardproject.dto.UserInfoResponseDto
-import com.kotlin.boardproject.dto.UserLoginMobileRequestDto
-import com.kotlin.boardproject.dto.UserLoginRequestDto
-import com.kotlin.boardproject.dto.UserSignUpDto
-import com.kotlin.boardproject.dto.UserSignUpMobileDto
-import com.kotlin.boardproject.dto.common.ApiResponse
-import com.kotlin.boardproject.repository.common.REFRESH_TOKEN
-import com.kotlin.boardproject.service.AuthService
+package com.kotlin.boardproject.domain.auth
+
+import com.kotlin.boardproject.domain.auth.dto.TokenResponseDto
```

Comment:
🚫 **[ktlint]** <standard:import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end